RELEASING.md

```diff
@@ -0,0 +1,18 @@
+# Release process notes
+
+## Prerelease
+
+- create a new branch with some name
+- write up changelog changes (manually), commit that
+- yarn lerna version --no-push 6.2.0-alpha.2
+- git push origin --follow-tags \$branch
+- make PR
+- merge PR
+- re-fetch the new master and check it out locally
+- yarn install
+- yarn lerna run prepare
```

Comment:
Similar to above
- yarn lerna run prepare
  - Runs the `prepare` script in each package directory (which does the compilation from ts to js)